rcode-reader/src/main/java/eu/rssw/pct/elements/BufferElement.java

```diff
@@ -0,0 +1,60 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.RCodeInfo;
+
+public class BufferElement extends AbstractAccessibleElement {
+  private static final int TEMP_TABLE = 4;
+
+  private String tableName;
+  private String databaseName;
+
+  private int flags;
+
+  public BufferElement(String name, Set<AccessType> accessType, byte[] segment, int currentPos, int textAreaOffset,
+      ByteOrder order) {
+    super(name, accessType);
+
+    int nameOffset = ByteBuffer.wrap(segment, currentPos, Integer.BYTES).order(order).getInt();
+    this.name = nameOffset == 0 ? "" : RCodeInfo.readNullTerminatedString(segment, textAreaOffset + nameOffset);
+
+    int tableNameOffset = ByteBuffer.wrap(segment, currentPos + 4, Integer.BYTES).order(order).getInt();
+    this.tableName = tableNameOffset == 0 ? ""
+        : RCodeInfo.readNullTerminatedString(segment, textAreaOffset + tableNameOffset);
+
+    int databaseNameOffset = ByteBuffer.wrap(segment, currentPos + 8, Integer.BYTES).order(order).getInt();
+    this.databaseName = databaseNameOffset == 0 ? ""
+        : RCodeInfo.readNullTerminatedString(segment, textAreaOffset + databaseNameOffset);
+
+    this.flags = ByteBuffer.wrap(segment, currentPos + 12, Short.BYTES).order(order).getShort();
+    // this.crc = ByteBuffer.wrap(segment, currentPos + 14, Short.BYTES).order(order).getShort();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)